backend/provider-service/src/jvmMain/kotlin/services/ProviderDao.kt

```diff
@@ -241,6 +240,7 @@ class ProviderDao(
     ): Boolean {
         val project = owner.project ?: return false
         if (actor == Actor.System) return true
+        if (actor.username == owner.createdBy) return true
```

Comment:
Should be removed. I know this will make the script slightly more complicated. The easiest solution in the script is probably to also make the user who owns `ADMINTOK` a member of the project. The username of `ADMINTOK` is `user1`.